common/backoff.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"time"
+	"github.com/cenkalti/backoff"
+)
+
+// Default values for PegExponentialBackOff.
+const (
+	DefaultInitialInterval     = 500 * time.Millisecond
+	DefaultRandomizationFactor = 0.5
+	DefaultMultiplier          = 1.5
+	DefaultMaxInterval         = 60 * time.Second
```

Comment:
a max interval of 60s doesn't make much sense while the max elapsed time is 10s. What about 2s?